pkg/cosign/verify.go

```diff
@@ -138,16 +144,39 @@ type ParamVerify struct {
 	CosignExePath      string
 }
 
-func (verifier *VerifierImpl) verify(ctx context.Context, param *ParamVerify) error {
+const tempErrMsg = "resource temporarily unavailable"
+
+var errVerify = errors.New("verify with Cosign")
+
+func (verifier *VerifierImpl) verify(ctx context.Context, logE *logrus.Entry, param *ParamVerify) error {
 	envs := []string{}
 	if param.CosignExperimental {
 		envs = []string{"COSIGN_EXPERIMENTAL=1"}
 	}
-	_, err := verifier.executor.ExecWithEnvs(ctx, verifier.cosignExePath, append([]string{"verify-blob"}, append(param.Opts, param.Target)...), envs)
-	if err != nil {
-		return fmt.Errorf("verify with cosign: %w", err)
+	for i := 0; i < 5; i++ {
+		// https://github.com/aquaproj/aqua/issues/1554
+		out, _, err := verifier.executor.ExecWithEnvsAndGetCombinedOutput(ctx, verifier.cosignExePath, append([]string{"verify-blob"}, append(param.Opts, param.Target)...), envs)
```

Comment:
🤔

out is empty. I'm investigating.

```
tlog entry verified with uuid: 962d963e758ef68ef09e52637a17dbc1f0a4f45fa7568cc387c9038fe8b3f898 index: 10085127
Verified OK
[DEBUG] 0 : 
```

Reply:
ref. https://github.com/suzuki-shunsuke/tfcmt/blob/bf4d2205253516f2cd289a6d73ec32605b55f4c3/pkg/controller/plan.go#L36-L46